src/app/view/web/WebChessView.ts

```diff
@@ -44,22 +44,16 @@ export class WebChessView implements ChessBoardView {
   }
 
   private renderPiecesOnBoard(piecesPositions: PiecesBoardPositions) {
-    let piecesSquareId: string;
-    let pieceImage: string;
-    let pieceName: string;
-    let pieceSide: string;
-
-    for (const piece in piecesPositions) {
-      if (piecesPositions.hasOwnProperty(piece)) {
-        pieceName = piecesPositions[piece].name.toLowerCase();
-        pieceSide = piecesPositions[piece].side.toLowerCase();
-
-        piecesSquareId = `#${piece.toLowerCase()}`;
-        pieceImage = `../static/img/pieces/${pieceSide}-${pieceName}.svg`;
-
-        this.parent.querySelector(piecesSquareId)?.appendChild(this.createPieceDiv(pieceImage, piece.toLowerCase()));
-      } else return;
-    }
+    Object.keys(piecesPositions)
+      .map((square) => {
+        const pieceName = piecesPositions[square].name.toLowerCase();
+        const pieceSide = piecesPositions[square].side.toLowerCase();
+        const pieceImage = `static/img/pieces/${pieceSide}-${pieceName}.svg`;
+        return { path: pieceImage, squareId: `#${square.toLowerCase()}`, square: square };
+      })
+      .forEach((element) => {
+        this.parent.querySelector(element.squareId)?.appendChild(this.createPieceDiv(element.path, element.square.toLowerCase()));
+      });
```

Comment:
@tomdworniczak a przecież my to zmienialiśmy wcześniej xD .. czyżbym coś źle kliknął jak robiłem pull developa i zaciągnąłem zmiany z tego poprzedniego taska? -.-

Reply:
dokładnie tak, musiałem to odgrzebać w commitach :P

Reply:
ok, to sorry... patrzyłem uważnie przy wszystkich, a jednak coś umknęło ... :-(

Reply:
luzik :D no harm done